website/docs/routes/picture.mdx

```diff
@@ -217,6 +217,20 @@ For example [Latest Wallpapers](https://wallhaven.cc/latest), the route turning
 
 <Route namespace="natgeo" data={{"path":"/dailyphoto","categories":["picture"],"example":"/natgeo/dailyphoto","parameters":{},"features":{"requireConfig":false,"requirePuppeteer":false,"antiCrawler":false,"supportBT":false,"supportPodcast":false,"supportScihub":false},"radar":{"source":["nationalgeographic.com/photo-of-the-day/*","nationalgeographic.com/"]},"name":"每日一图","maintainers":["LogicJake","OrangeEd1t","TonyRL"],"url":"nationalgeographic.com/photo-of-the-day/*","location":"dailyphoto.ts"}} />
 
+## 极品性感美女
+
+### 本周热门
+
+<Route namespace="jpxgmn" data={{"path":"/weekly","categories":["picture"],"example":"/jpxgmn/weekly","features":{"requireConfig":false,"requirePuppeteer":false,"antiCrawler":false,"supportBT":false,"supportPodcast":false,"supportScihub":false},"name":"本周热门","maintainers":["Urabartin"],"location":"weekly.ts"}} />
+
+### 分类
+
+<Route namespace="jpxgmn" data={{"path":"/tab/:tab?","categories":["picture"],"example":"/jpxgmn/tab","parameters":{"tab":"分类，默认为`top`，包括`top`、`new`、`hot`，以及[源网站](http://www.jpxgmn.com/)所包含的其他相对路径，比如`Xiuren`、`XiaoYu`等"},"features":{"requireConfig":false,"requirePuppeteer":false,"antiCrawler":false,"supportBT":false,"supportPodcast":false,"supportScihub":false},"name":"分类","maintainers":["Urabartin"],"location":"tab.ts"}} />
+
+### 搜索
+
+<Route namespace="jpxgmn" data={{"path":"/search/:kw","categories":["picture"],"example":"/jpxgmn/search/candy","parameters":{"kw":"搜索关键词"},"features":{"requireConfig":false,"requirePuppeteer":false,"antiCrawler":false,"supportBT":false,"supportPodcast":false,"supportScihub":false},"name":"搜索","maintainers":["Urabartin"],"location":"search.ts"}} />
+
```

Comment:
Do not modify the docs, it will be generated automatically.

Reply:
done